spec/support/system_helpers.rb

```diff
@@ -16,4 +18,8 @@ def form_label(key)
   def css_id(record)
     "##{dom_id(record)}"
   end
+
+  def frontend_translations
+    JSON.parse(FRONTEND_TRANSLATIONS)
```

Comment:
This will still be re-parsed on every access… maybe memoize it?
```suggestion
    @@frontend_translations ||= JSON.parse(FRONTEND_TRANSLATIONS)
```